apps/fhir/bluebutton/utils.py

```diff
@@ -595,7 +595,6 @@ def build_oauth_resource(request, v2=False, format_type="json"):
     """
     endpoints = build_endpoint_info(OrderedDict(), v2,
                                     issuer=base_issuer(request))
-    logger.info("\nEndpoints:%s" % endpoints)
```

Comment:
Removing this old INFO logging, since it causes JSON parsing issues. I think this might have been used for debugging and not removed afterward. There is other logging related to the request/response, so is OK to remove.

Below is the resulting log entry for this and any other JSON parsing issues will be similar:

```
{
  "time_of_event": "2021-12-02T19:18:53",
  "component": "bb2.web",
  "vpc": "UNKNOWN-JSON-DECODE-ERROR",
  "log_name": "UNKNOWN-JSON-DECODE-ERROR",
  "cw_log_event_id": "36539162933373097684885866129210360744161566069418295300",
  "type": "bfd-insights-log-lambda-error",
  "mesg": "Warning message_dict is None. Could not parse original JSON message.",
  "log_event": {
    "id": "36539162933373097684885866129210360744161566069418295300",
    "timestamp": 1638472733000,
    "message": "Endpoints:OrderedDict([('issuer', 'https://sandbox.bluebutton.cms.gov'), ('authorization_endpoint', 'https://sandbox.bluebutton.cms.gov/v1/o/authorize/'), ('token_endpoint', 'https://sandbox.bluebutton.cms.gov/v1/o/token/'), ('userinfo_endpoint', 'https://sandbox.bluebutton.cms.gov/v1/connect/userinfo'), ('ui_locales_supported', ['en-US']), ('service_documentation', 'https://bluebutton.cms.gov/developers'), ('op_tos_uri', 'https://bluebutton.cms.gov/terms'), ('grant_types_supported', ['authorization-code', 'refresh_token']), ('response_types_supported', ['code', 'token']), ('fhir_metadata_uri', 'https://sandbox.bluebutton.cms.gov/v1/fhir/metadata')])}"
  }
}

```